### ISSUE TITLE
Win/draw% values are misleading

### ISSUE THREAD

**Issue by craylton** [TCEC-Chess/tcecgui#41]
This is about the `21.2% W | 78.8% D` values displayed next to the engine names and scores.
The way they're calculated assumes that there are only 2 possible outcomes - win or draw. E.g. if the eval is 0.10, then the probability of black winning is absolutely zero.
Even worse, if the eval is 0.00, then the probability of a draw is calculated as 100%. If we are 100% sure of the outcome of the game, why bother even playing it?
Since these values aren't realistic most of the time, I see them as just UI clutter. So the way I see it, either they should be removed, or the calculations improved to estimate all 3 possible outcomes (I have equations for this if interested).

**Comment by TerjeKir** [comment]
I support having them removed (at least as an option) :)

**Comment by xoto10** [comment]
See Options - Info - Info Percent - Off

**Comment by craylton** [comment]
Ah, the new GUI is much more configurable than I realised :)
The reason I made this issue in the first place though is because I was seeing multiple people in chat who were confused by those those values. The values are visible by default, and after playing with the options I can see there is even an option to graph the % score instead of cp. So I don't think it should be ignored.
I've done some quick analysis and found that the current equations have an error rate of 26.2% across the main events in seasons 15 16 & 17. This is only slightly better than if you simply return W:15% D:80% B:5% regardless of eval, which gives an error of 30.9%. This can easily be improved, I've tweaked an equation that gives an 18.7% error for example.
